src/desktop/apps/auction/queries/me_v2.js

```diff
@@ -0,0 +1,48 @@
+export default function MeQueryV2(sale_id, live = true) {
```

Comment:
is the `live` argument needed? Seems like we don't [relevant query below](https://github.com/artsy/force/pull/5401/files#diff-f1120c85eda36464dccd49e43728e8a9R67)